cts.json

```diff
@@ -2174,6 +2174,15 @@
       "selector" : "$[?@.a[*].a==0]",
       "invalid_selector": true
     },
+    {
+      "name": "filter, nested",
+      "selector": "$.a[?@>count($.b[?@>5])]",
```

Comment:
Is there another way to make a nested query that doesn't include a function?  We only want to test one thing here.  This tests

- nested queries
- (non-singular) queries as function args
- `count()` function

Reply:
How about `$[?@[?@.a=='foo']]`?  In an array of arrays, this would return any array that contained an object that contains an `a` property with a value of "foo".

Data:

```json
[
  [1, {"a": "foo"}],
  [2, {"b": "foo"}],
  [3. {"a": "bar"}]
]
```

Result:
```json
[
  [1, {"a": "foo"}]
]
```

Reply:
Good point, will update.